_components/header/header.html

```diff
@@ -92,85 +86,52 @@ <h2>Header usability</h2>
-
+    <strong>Add a skip navigation link before the header.</strong> Include skip navigation links to allow users with
+    screen readers to bypass long navigation lists. Make sure you include an <code>id</code> at the beginning of your
+    main content and that it matches the skip navigation link. Find more <a
+      href="http://webaim.org/techniques/skipnav/">information on these links</a> at WebAIM.
   </li>
   <li>
-    Include tab focus for all top-level navigation navigation items — this feature will
-    allow keyboard-reliant users to easily navigate interactive items.
+    <strong>Ensure your horizontal navigation is keyboard compatible.</strong> Test to make sure
+    users can use <code>Tab</code> to navigate and <code>Space</code> (or <code>Enter</code>) to open pages.
   </li>
   <li>
-    Ensure your horizontal navigation is keyboard compatible; test to make sure
-    users can use Tab to navigate and Space (or Enter) to open pages.
+    <strong>Include tab focus for all top-level navigation navigation items.</strong> This feature will
+    allow keyboard-reliant users to easily navigate interactive items.
   </li>
   <li>
-    Avoid using hover to expand dropdown lists. Hover is difficult for some
+    <strong>Avoid using hover to expand dropdown lists.</strong> Hover is difficult for some
     users and won't work on touch screens. Dropdowns should expand on click or
     with keyboard navigation.
   </li>
   <li>
-    The Design System uses semantic header and nav elements with
-    <code>role="banner"</code> and <code>role="navigation"</code> respectively.
-    <code>role="banner"</code> is your masthead.
-  </li>
-  <li>
-    You can use multiple nav elements for groups of navigation links, but you
-    should only use one <code>role="navigation"</code> for the main nav of a
-    page.
+    <strong>Add context by labelling your nav element.</strong> If your page has more than one
+    <code>nav</code> element, use the <code>aria-label</code> attribute to help assistive technology users understand the purpose of the navigation.
   </li>
   <li>
-    Use lists for your nav links — doing so enables screen readers to decipher
-    header contents.
+    <strong>Use list elements for your navigation links.</strong> This allows screen readers users to more easily
+    navigate header content.
   </li>
   <li>
-    If you’re using a logo that’s an image rather than text, make sure you
-    include alternative text for screen readers.
+    <strong>Add alt text to your logo image.</strong> If you’re using a logo that’s an image rather than text, make sure
+    you include alternative text for screen readers.
   </li>
   <li>
-    If you’re using a logo that’s text, use an <code>em</code>, not an
+    <strong>Don't use an H1 for your logo.</strong> If you’re using a logo that’s text, use an <code>em</code>, not an
```

Comment:
Is the bolded statement accurate?

Reply:
Reflects what's shown in example:
<img width="382" alt="image" src="https://github.com/uswds/uswds-site/assets/3385219/0fa4d991-b054-4817-a8e8-28d5535c49de">

[Default header preview](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.sites.pages.cloud.gov/preview/uswds/uswds/develop/?path=/story/components-header--default)